### DISCUSSION TITLE
There is no payout info after the update

### DISCUSSION THREAD

**Discussion by fisch88** [martomi/chiadog#277] [Q&A]
Before the version update, as soon as I found a win I got a message: "Found 1 proof (s)" from the harvester chiadog.
Now with the new version I get "Block found" but I get this from the FULL NODE without knowing where the win is coming from.
Is there any setting?
Tnx

**Comment by mazm786** [comment]
Waiting for reply to your question from any relevant person. Have you found your answer ?
